app/assets/javascripts/admin/preview.js

```diff
@@ -1,36 +1,37 @@
+/* globals govspeakBarcharts */
```

Comment:
Why are some of these `global` and some are `globals`?

Reply:
Oh dear - I didn't even realise, thanks for catching. I've set them all to global now - it's a bit of a shame it accepts both.